src/repositories/sql/users/fetch_profile.sql

```diff
@@ -0,0 +1,20 @@
+SELECT * FROM users
+JOIN usertypes ON users.usertype_id=usertypes.usertype_id WHERE user_id=$1;
+
+SELECT user_id,
+B.usertype_title,
+username,
+CASE WHEN show_name=FALSE THEN NULL
+        ELSE given_name
+        END,
+CASE WHEN show_name=FALSE THEN NULL
+        ELSE family_name
+        END,
+CASE WHEN show_email=FALSE THEN NULL
+        ELSE email
+        END,
+CASE WHEN show_contact=FALSE THEN NULL
+        ELSE contact
+       END
+FROM users
+A INNER JOIN usertypes ON A.usertype_id=B.usertype_id WHERE user_id=$1;
```

Comment:
Thank you. This one looks good so far, but is still missing the required fields.

From #20 
> - Create a `SELECT` command to fetch a user from `users` when given a `user_id`, return only the following fields:
>   - **user_id**
>   - **usertype_title** from `usertypes`
>   - **username**
>   - **given_name**, return `null` if `show_name` is `false`
>   - **family_name**, return `null` if `show_name` is `false`
>   - **email**, return `null` if `show_email` is `false`
>   - **contact**, return `null` if `show_contact` is `false`
>   - **avatar_image**
>   - **background_image**
>   - **created_at**


For the next screenshot, please set your sample user's `show_name` to `TRUE`. Upload the screenshot in a new comment and include the following:
- Your sample `user` 
- The result of this query after the requested changes